static/js/util/util.js

```diff
@@ -46,10 +47,27 @@ function asPercent(number) {
  */
 export function makeCourseStatusDisplay(course, now = moment()) {
   let courseOrRun = course;
+  let courseUpgradeUrl = [];
+  let courseInfoUrl = [];
+
   if (course.runs.length > 0) {
     courseOrRun = course.runs[0];
   }
 
+  courseInfoUrl.push(
```

Comment:
What advantage is there to using `push` here? Since `join` is just simple string concatenation, can you use a ES6 template string here:

    let courseInfoUrl = `${SETTINGS.edx_base_url}courses/${courseOrRun.course_id}/info`;